target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/vr9_avm_fritz7430.dts

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7430", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7430";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &info_red;
+		led-running = &power;
+		led-upgrade = &info_red;
+
+		led-dsl = &info_green;
+		led-wifi= &wifi;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		dect {
+			label = "dect";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_PHONE>;
+		};
+
+		wifi {
+			label = "wifi";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WLAN>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power: power {
```

Comment:
Please prefix the DT label (as it's global) with "led_":

`led_power: power {`

Same below.